### PR TITLE
Create sPHENIX VO. 

### PR DESCRIPTION
**Pull request by jhover**
Needs ID assigned to sPHENIX VO.

### CODE REVIEW

File: virtual-organizations/sPHENIX.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  VO Manager:
+  - ID: acbb79b1cef1d22d7ca443ccd1aa9556750a8f3f
+    Name: Christopher Pinkenburg
+Disable: false
+FieldsOfScience:
+  PrimaryFields:
+  - Nuclear Physics
+  SecondaryFields:
+  - Physics and astronomy
+ID:
```
Comment (matyasselmeci) [inline virtual-organizations/sPHENIX.yaml]:
```suggestion
ID: 137
```

**Comment by jhover** [comment]
So I should add the ID, or do you do it when you merge?

--john

**Comment by matyasselmeci** [comment]
Please add it so that the CI tests rerun.

**Comment by matyasselmeci** [comment]
Looks good. Sorry about editing your master branch.

**Comment by jhover** [comment]
No problem. Whatever works.

Just realized about the 'suggested edits' thing in the PR message
thread. Now I'll just look for that.

-j

**Pull request merged**